timeless/templates/reservations/create_edit.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'base.html' %}
+{% from "_formhelpers.html" import render_field %}
+
+{% block header %}
+  <h1>{% block title %}Reservation management - {% if action == 'create' %}  Create {% else %} Edit {% endif %} {% endblock %}</h1>
+{% endblock %}
+
+{% block content %}
+  <form method="post">
+    {{ form }}
+    <p>
+      <button type="submit">Create</button>
+    </p>
+  </form>
+{% endblock %}
```

Comment:
@Vatavuk new line here. And why did you delete the ``create_edit.html`` file bellow?

Reply:
@amihaiemil I've moved it to the reservation directory